### ISSUE TITLE
Remove the option to disable pump unreachable alerts

### ISSUE THREAD

**Issue by bartsopers** [AAPS-Omnipod/AndroidAPS-OLD#59]
As we have fixed Pump unreachable alerts, the option to disable alerts doesn't have any effect. We should remove the option for pumps with fixed unreachable alerts in PreferenceFragment:

```
if(activePump.hasFixedUnreachableAlerts) {
    preferenceScreen.removePreference(findPreference("enable_pump_unreachable_alert"))
}
```

**Comment by bartsopers** [comment]
The above solution only works when the dagger branch has been merged, which will probably happen in the 2.7 release. We should either wait for that, or fix it in a different way in the current code.

**Comment by bartsopers** [comment]
fixed in 6f2e39dca7eee3d8bac47f23d7f42e68a3d62fd4

**Comment by bartsopers** [comment]
Did some improvements in bc06b6b24a9d03d18e1c76ac6bad0c53ad138687:

- Don't remove the 'Enable pump unreachable alerts' preference for pumps with fixed unreachable alerts, but disable it instead. 
- Make the threshold for the unreachable alerts configurable for pumps with fixed unreachable alerts. (In my initial implementation of fixed unreachable alerts, it just used a hardcoded value of 30 minutes, and ignored the threshold setting. In the previous commit mentioned in this issue, I still used the hardcoded value of 30 minutes, and removed the 'Pump unreachable treshold' setting for pumps with fixed unreachable alerts. In this commit, the threshold setting is restored and actually taken into account)
